db/schema.rb

```diff
@@ -189,6 +189,23 @@
     t.index ["track_owner_id"], name: "index_listens_on_track_owner_id"
   end
 
+  create_table "mass_invite_signups", charset: "utf8mb4", force: :cascade do |t|
```

Comment:
I have the `utf8mb4_unicode_ci` specified locally and everywhere in the schema but *not* in production, for some reason (perhaps it's default there?)

So I have some diff that looks like so:

```ruby 
create_table "mass_invites", charset: "utf8mb4", collation: "utf8mb4_unicode_ci", force: :cascade do |t|
```

This is just an informational notice, I don't really remember why things are this way, I will come back and look to see if it matters.

Reply:
Context in https://github.com/sudara/alonetone/issues/732#issuecomment-532141963

Reply:
It matters when you sort text columns without specifying the collation or do full text search, otherwise it doesn't really matter. But I may be missing something.

Reply:
Past me is a lot smarter than current me.

Reply:
I'm still on MySQL 5.7 for a project to match production. We had the same inconsistency there, so it's either caused by my workstation setup or a difference in MySQL version.